packages/react-scripts/template/src/vendors.js

```diff
@@ -0,0 +1 @@
+module.exports = ['react', 'react-dom'];
```

Comment:
How about we add a comment here explaining what adding an entry to this list does?

```js
// This file controls which npm packages are extracted into a vendor bundle.
// A vendor bundle is very useful for reducing your transmitted bundle size, as 
// dependencies which rarely change can be cached while you iterate on your application.
// (edit this comment as you see fit and make it run to 80 cols please)

module.exports = ['react', 'react-dom'];

// You can delete this file if you'd prefer to not make a vendor bundle.
// (edit this comment as you see fit and make it run to 80 cols please)
```